templates/vulnerability.sample

```diff
@@ -0,0 +1,481 @@
+{ 
```

Comment:
No need to add the whole json here? just the relevant part to build the vulnerability may be enough?

Something like:
```
{ 
   "title":"qTranslate 2.5.34 - Setting Manipulation CSRF",
    "fixed_in":null,
    "references":{ 
       "cve":[ 
          "2013-3251"
        ],
       "wpvulndb":[ 
          "6846"
        ]
     }
  }
```

Reply:
I could trim it down, but I think the sample file also helps in testing, and the more real world it is the better for testing purposes. But I really don't mind either way.